### PR TITLE
Implement NSIS installer for Lime3DS

### LINKED ISSUE #297: Distribute Lime3DS via an installer on Windows
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

I want to be able to install and update Lime3DS via an installer.

### Why would this feature be useful?

That would make it user friendly to update and possible to update via e.g. winget.

### PR DESCRIPTION
**Pull request by OpenSauce04**
Closes #297

This pull request adds an installer for Lime3DS which will be distributed with releases from now on instead of the previously used zip file.

This installer is based on Dolphin's NSIS install script, and is credited as such.

Additionally, the old data for Citra's installer has been gutted as it no longer serves any purpose.

I have created a test release on my fork to verify the functionality of the generated installers: https://github.com/OpenSauce04/Lime3DS/releases/tag/1234

### CODE REVIEW

**Comment by DavidRGriswold** [comment]
I just tested both windows versions. installed both, launched it, uninstalled. No problems. I don't actually have any 3ds games on my windows laptop right now so I didn't test functionality, but I assume that shouldn't matter. 

I have not tested what happens if you try to install the MSVC without the .NET requirements.

I also looked through the code, it seems fine, but I know nothing about NSIS so it's hard for me to know if anything in the config is unnecessary. But nothing stuck out.

**Comment by OpenSauce04** [comment]
> I have not tested what happens if you try to install the MSVC without the .NET requirements.

The MSVC installer itself does not have the same requirements as the Lime3DS program. Attempting to install it without the dependencies will work fine, but running the program will have the same result as the older zip file distributions, and can be resolved in the same way.

File: src/installer/lime3ds.nsi [hunk 1]
```diff
@@ -0,0 +1,201 @@
+; Copyright Dolphin Emulator Project
```
Comment (DavidRGriswold) [inline src/installer/lime3ds.nsi]: Should this be removed or changed? I know most of this code comes from there, but I assume parts are different.

**Review by DavidRGriswold — COMMENTED** [review]
Should this copyright notice be tweaked? Otherwise, I'm ready to approve.

File: src/installer/lime3ds.nsi (hunk 1 above)
Comment (OpenSauce04) [reply src/installer/lime3ds.nsi]: On second look, it should probably also mention Lime3DS as well. A lot of the file was changed.

**Review by DavidRGriswold — APPROVED** [review]
looks good to me.

**Review by DavidRGriswold — APPROVED** [review]
Merge away as far as I'm concerned.

**Pull request merged**